tests/cli/plugin/test_command.py

```diff
@@ -0,0 +1,15 @@
+import shutil
+
+import importlib_resources
+
+
+def test_run_command(hatch, temp_dir, helpers):
+    project_dir = temp_dir / "cli-plugin"
+    shutil.copytree(importlib_resources.files(__package__) / "cli-plugin", project_dir)
+
+    with project_dir.as_cwd():
+        result1 = hatch("new", "--init")
```

Comment:
```suggestion
        hatch("new", "--init")
```

Reply:
This is still in draft, that part is anyway not working (cf: https://github.com/pypa/hatch/pull/805#issuecomment-1493134366) -> forgot to add a `TODO` on top